src/internal/modules/charge-information/pre-handlers.js

```diff
@@ -107,6 +107,17 @@ const loadLicencesWithWorkflowsInProgress = async request => {
   }
 };
 
+const loadChargeVersions = async request => {
+  const { licenceId } = request.params;
+  try {
+    const { data: chargeVersions } = await services.water.chargeVersions.getChargeVersionsByLicenceId(licenceId);
+    return chargeVersions.filter(cv => (cv.status === 'current' || cv.status === 'superseded'))
+      .sort((rowA, rowB) => new Date(rowB.dateRange.startDate) - new Date(rowA.dateRange.startDate));
```

Comment:
Looks like charge versions are already filtered in above code, so filtering may not be needed here.
I generally sort by start date then version number, so this might become:

```
const { sortBy } = require('lodash');
// ...
return sortBy(chargeVersions, ['dateRange.startDate', 'versionNumber']);
```